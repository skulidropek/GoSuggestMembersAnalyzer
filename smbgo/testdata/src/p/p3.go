package p

import _ "example.com/testmod/storag" // want "cannot resolve import"
