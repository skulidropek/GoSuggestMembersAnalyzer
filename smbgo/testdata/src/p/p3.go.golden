package p

import _ "example.com/testmod/storage" // want "cannot resolve import"
